lib/screens/portfolio/coin_detail/tx_list_item.dart

```diff
@@ -24,6 +27,11 @@ class _TransactionListItemState extends State<TransactionListItem> {
   CexProvider cexProvider;
   bool isNoteExpanded = false;
 
+  double get transactionValue =>
+      double.parse(widget.transaction.myBalanceChange);
+
+  bool get isWarningTransaction => transactionValue == 0;
```

Comment:
Since only EVM networks are affected, it probably makes sense to also check the protocol, what do you think?

Reply:
Thanks, @yurii-khi. Is there a legitimate use case for zero-value trades where we wouldn't add it to the list of networks we're checking if we noticed a similar pattern on another network?

Has either you or the API team defined standard criteria? I'm happy to follow that since it's important we're consistent across all platforms.

Reply:
https://github.com/KomodoPlatform/atomicdex-mobile/blob/373eb15cca17b6669fe7c114181678622e4b9a92/lib/screens/portfolio/coin_detail/tx_list_item.dart#L33-L34